core/domain/takeout_service_test.py

```diff
@@ -448,7 +449,7 @@ def test_export_nonexistent_user(self):
         """Setup for nonexistent user test of export_data functionality."""
         with self.assertRaises(
             user_models.UserSettingsModel.EntityNotFoundError):
-            takeout_service.export_data_for_user('fake_user_id')
+            takeout_service.export_data_for_user('fake_user')
```

Comment:
Why change this from what it was before? Usually a user ID is passed in, right?

Reply:
Done